tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/SolutionRepositoryTests.cs

```diff
@@ -114,11 +108,6 @@ await CatalogueItemEntityBuilder
             await SolutionEntityBuilder.Create()
                 .WithId(Solution1Id)
                 .Build()
-                .InsertAsync();
-
-            await SolutionDetailEntityBuilder.Create()
-                .WithSolutionId(Solution1Id)
-                .Build()
                 .InsertAndSetCurrentForSolutionAsync();
```

Comment:
As above, we should call `InsertAsync` instead of `InsertAndSetCurrentForSolutionAsync`.

Reply:
Replaced `InsertAndSetCurrentForSolutionAsync` with `InsertAsync`